walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -1,25 +1,115 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
-}
+@media screen and (min-width: 600px)
```

Comment:
Having all of your styles wrapped in a media query means that if these conditions aren't met your entire page will be unstyled. Typically css is styled for desktop devices outside of media queries, and styled for mobile/tablet/etc in media queries (although this is not always the case).

Reply:
I moved the styling to outside the media query, will use media query in the future so left comment.